lib/FullScreenView/tests/FullScreenView-test.js

```diff
@@ -36,35 +38,57 @@ describe('FullScreenView', () => {
   describe('rendering FullScreenView', () => {
     const paneTitle = 'Pane title';
     const handleCancel = sinon.spy();
+    const renderHeader = sinon.spy();
 
-    beforeEach(async () => {
-      handleCancel.resetHistory();
+    describe('not passing renderHeader prop', () => {
```

Comment:
`rendering FullScreenView without passing renderHeader prop`
```suggestion
    describe('rendering FullScreenView without renderHeader prop', () => {
```